src/libraries/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -31,10 +31,10 @@
     <Compile Include="System\Net\Security\SslClientAuthenticationOptions.cs" />
     <Compile Include="System\Net\Security\SslClientHelloInfo.cs" />
     <Compile Include="System\Net\Security\SslServerAuthenticationOptions.cs" />
-    <Compile Include="System\Net\Security\SecureChannel.cs" />
     <Compile Include="System\Net\Security\SslSessionsCache.cs" />
     <Compile Include="System\Net\Security\SslStream.cs" />
-    <Compile Include="System\Net\Security\SslStream.Implementation.cs" />
+    <Compile Include="System\Net\Security\SslStream.IO.cs" />
+    <Compile Include="System\Net\Security\SslStream.Protocol.cs" />
```

Comment:
I'm curious why we want to continue splitting it at all?  Is it really unmanageable in a single file?

Reply:
We probably can if that is preference. The "SslStream.Implementqation" was always somewhat weird to me. I think we can be careful nad use sections to make it more manageable - at least in VS. 
Do you want me to do it as part of this PR?

Reply:
It seems like preserving history _may_ be possible but tricky. I can look into it as separate PR. (rename just work)